indico/modules/events/sessions/templates/management/_types_table.html

```diff
@@ -0,0 +1,18 @@
+{% macro types_table_row(session_type) %}
```

Comment:
This is a duplicate of what we have for contribution types. Maybe it would be worth it to extract it to a more generic template that could be used in different places?

Reply:
Not sure if it's really worth the effort or even makes much sense since the poster column is only used here, and there are quite a few references to "session types" in other lines (e.g. confirmation messages), so the amount of reusable code isn't that big.

Reply:
I agree with @ThiefMaster At first I thought about making some parts more generic, but I figured it's not really worth complicating.